packages/repository-json-schema/src/filter-json-schema.ts

```diff
@@ -101,6 +101,10 @@ export function getFilterJsonSchemaFor(
           : undefined,
       type: 'array',
       items: {
+        title:
+          options.setTitle !== false
+            ? `${modelCtor.modelName}.IncludeFilter.Items`
+            : undefined,
```

Comment:
It would be ideal to not even add an `undefined` title property if `setTitle === false`.

Reply:
How about below?  Code is harder to read but more concise.

@bajtos Would this be OK?

Amended:
```ts
  {
    ...(options.setTitle !== false && {
      title: `${modelCtor.modelName}.Filter`,
    }),
  }
```
From:
```ts
  {
    title:
      options.setTitle !== false ? `${modelCtor.modelName}.Filter` : undefined,
  }
```